app/src/main/java/helium314/keyboard/latin/suggestions/SuggestionStripView.java

```diff
@@ -445,12 +446,7 @@ private void showSourceDict(final TextView wordView) {
         final SuggestedWordInfo info = mSuggestedWords.getInfo(index);
         if (!info.getWord().equals(word)) return;
         final String text = info.mSourceDict.mDictType + ":" + info.mSourceDict.mLocale;
-        // apparently toast is not working on some Android versions, probably
-        // Android 13 with the notification permission
-        // Toast.makeText(getContext(), text, Toast.LENGTH_LONG).show();
-        final PopupMenu uglyWorkaround = new PopupMenu(DialogUtilsKt.getPlatformDialogThemeContext(getContext()), wordView);
-        uglyWorkaround.getMenu().add(Menu.NONE, 1, Menu.NONE, text);
-        uglyWorkaround.show();
+        mListener.showToast(text, true, true);
```

Comment:
For me, the toast is hidden behind the more suggestions panel. Is it the same for you?

Reply:
I just checked and it is also hidden for me
There should be some way to make the toast view appear on top
Alternatively another 'ugly workaround' would be to dismiss the more suggestions panel just before the toast would get shown
In landscape mode the fake toast isn't shown even when the panel is dismissed, but mFakeToast.bringToFront() seems to solve that